app/models/enclosure_url_builder.rb

```diff
@@ -21,8 +21,17 @@ def podcast_episode_url(podcast, episode, feed = nil)
     feed ||= podcast.default_feed
     prefix = feed.try(:enclosure_prefix)
 
-    url = base_enclosure_url(podcast, episode, feed)
-    enclosure_prefix_url(url, prefix)
+    base_url = if !episode.enclosure_override_url.blank?
+      episode.enclosure_override_url
```

Comment:
So this will return the override url, regardless of its status.  Are there protections against serving a processing/invalid one?  Or TBD?

Reply:
no protection -  if you override, it's overidden - when we are talking about this being used for megaphone, there is no condition where it would be better to serve the underlying media instead (unless the override was removed).

Reply:
protection for this will come from the publishing pipeline, which won't publish the episode until the override is processed

Reply:
I was thinking more about what would happen if `medium = override`, the episode was published, and you replaced/changed the enclosure.

I don't want to get too wrapped up in that edge case though, so this seems a good place to start!